rebasehelper/cli.py

```diff
@@ -249,10 +257,11 @@ def run():
                 raise RebaseHelperError('Wrong format of --builder-options. It must be in the following form:'
                                         ' --builder-options="--desired-builder-option".')
             cli = CLI()
-            execution_dir, results_dir, debug_log_file, report_log_file = Application.setup(cli)
+            execution_dir, results_dir, debug_log_file = Application.setup(cli)
             if not cli.verbose:
                 handler.setLevel(logging.INFO)
-            app = Application(cli, execution_dir, results_dir, debug_log_file, report_log_file)
+            app = Application(cli, execution_dir, results_dir, debug_log_file)
+            ConsoleHelper.should_use_colors(app)
```

Comment:
I think this should be called sooner, perhaps even in `CLI` constructor. We might want to print some colorized messages in `Application.setup` or so.